src/target/source/codegen_c.cc

```diff
@@ -662,6 +662,11 @@ void CodeGenC::VisitExpr_(const CallNode* op, std::ostream& os) {  // NOLINT(*)
       os << " != ";
       this->PrintExpr(op->args[0], os);
       os << ")";
+    } else if (op->op.same_as(builtin::lookup_param())) {
+      ICHECK_EQ(op->args.size(), 1);
+      const StringImmNode* str = op->args[0].as<StringImmNode>();
+      ICHECK(str != nullptr);
+      os << "__tvm_param__" << str->value;
```

Comment:
along the lines of @mjs comment that `__` are reserved for c++ compiler naming, shall we rename this to simply `tvm_param_`? and, we should consider naming this after the model as well. finally, it would be great to retain `tvm_param_prefix` here

Reply:
I am not mangling here, because those parameters are static, hence only visible internally in the compilation unit (and not in the library). I can mangle them, but it means that the code generator needs to know the `mod_name`, so I need to hack the `Codegen` method etc.. I think @mshawcroft was referring to  names exposed in the library (operators name, runner name, etc..) as opposed to local names to each compilation unit